pkg/config/config.go

```diff
@@ -228,17 +229,24 @@ func (c *configImpl) replaceDependencies(data map[string]map[string]string, dict
 }
 
 func (c *configImpl) parseDependency(dependency string, dict map[string]api.DynatraceEntity) (string, error) {
+	absolutePath := false
```

Comment:
can we not call this `absolutePath` ? These are still relative/absolute only from the project root, aren't they?

Reply:
Yeah but they are absolute from the project root. But yeah, if you have a better
word for it, I'm open for suggestions :+1:

Reply:
`relativelyAbsolutePath` :P 
as we have general issues with absolute and relative path handling I'd prefer not adding confusion by naming things wrong

Reply:
What is the official name for such references? 
We were always talking about absolute paths, so this kinda burned into
my head. But if you tell me that we actually call it something else, then 
yeah sure :laughing:

Reply:
"official name" as in official in our concept? 
"officially" relative and absolute paths are pretty clear cut in their definitions, and I wouldn't want us to add confusing terminology to the already confusing (thus currently very brittle) handling of paths in monaco

I guess in this case it's `inAbsolutePathFormat` - but monaco does not use it (or even function) if it's an actual absolute path from os root /

Reply:
I like `absoluteFromProjectRoot`! :+1:

Reply:
I renamed the variable to `absoluteFromProjectRoot`